macros/compare_queries.sql

```diff
@@ -106,5 +106,9 @@ final as (
 {%- endif %}
 
 select * from final
+{%- if not summarize and limit %}
```

Comment:
```suggestion
{%- if limit and not summarize %}
```

This suggestion is equivalent, but I first read the original version as `not (summarize and limit)` instead of the correct version, `(not summarize) and limit` so this makes it a bit clearer.